qaframework-legacy-tests/src/test/java/org/openmrs/contrib/qaframework/legacy/RegisterPatientTest.java

```diff
@@ -34,7 +33,7 @@
     @Category(BuildTests.class)
     public void registerPatientTest() throws InterruptedException {
         RegistrationPage registrationPage = homePage.goToRegisterPatientApp();
-        patient = PatientGenerator.generateTestPatient();
+        patient = TestData.PatientGenerator.generateTestPatient();
```

Comment:
Well done , cross check with in this class and correct the comment, Pul them on same time if possible. 
Forexample [here](https://github.com/openmrs/openmrs-contrib-qaframework/blob/a9a7aa0738219ae6d1dcd7b05a8dc42641d3f72c/qaframework-bdd-tests/src/main/java/org/openmrs/contrib/qaframework/helper/TestData.java#L157)